tools/pot/docs/BestPractices.md

```diff
@@ -73,15 +73,15 @@ For example, the following `range_estimator` for activations can improve the acc
 Find the possible options and their description in the `configs/default_quantization_spec.json` file in the POT directory.
 
 4.  The next option is `stat_subset_size`. It controls the size of the calibration dataset used by POT to collect statistics for quantization parameters initialization.
-It is assumed that this dataset should contain a sufficient number of representative samples. Thus, varying this parameter may affect accuracy (higher is better). 
-However, we empirically found that 300 samples are sufficient to get representative statistics in most cases.
-5.  The last option is `ignored_scope`. It allows excluding some layers from the quantization process, i.e. their inputs will not be quantized. It may be helpful for some patterns for which it is known in advance that they drop accuracy when executing in low-precision.
+It is assumed that this dataset should contain a sufficient number of representative samples. Hence, varying this parameter may affect accuracy (higher is better). 
+However, it shows that 300 samples are sufficient to get representative statistics in most cases.
+5.  The last option is `ignored_scope`. It allows excluding some layers from the quantization process, for example, their inputs will not be quantized. It may be helpful for some patterns, which are known in advance that they drop accuracy when executing in low-precision.
```

Comment:
```suggestion
5.  The last option is the `ignored_scope`. It allows excluding some layers from the quantization process, for example, their inputs will not be quantized. It may be helpful for some patterns, which are known in advance, that they drop accuracy when executing in low-precision.
```